packages/forms/src/styled/select/StyledSelectWrapper.ts

```diff
@@ -5,13 +5,21 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
+import styled, { DefaultTheme, ThemeProps, css } from 'styled-components';
 import { DEFAULT_THEME, SELECTOR_FOCUS_VISIBLE } from '@zendeskgarden/react-theming';
-import { StyledTextFauxInput } from '../text/StyledTextFauxInput';
+import { IStyledTextFauxInputProps, StyledTextFauxInput } from '../text/StyledTextFauxInput';
 import { StyledSelect } from './StyledSelect';
 
 const COMPONENT_ID = 'forms.select_wrapper';
 
+const sizeStyles = ({ theme, isCompact }: IStyledTextFauxInputProps & ThemeProps<DefaultTheme>) => {
+  const height = `${theme.space.base * (isCompact ? 8 : 10)}px`;
+
+  return css`
+    max-height: ${height};
```

Comment:
Fixes a 2px height bump difference between `Select` and `Input` due to wrapper border.

Reply:
Yeah. In general, I'm really looking to getting rid of all this faux/wrapper/media/input differentiation in v9 with a new component.next that rolls everything into one.